ocs_ci/deployment/deployment.py

```diff
@@ -765,6 +769,8 @@ def deploy_cluster(self, log_cli_level="DEBUG"):
             from ocs_ci.deployment.hosted_cluster import enable_nested_virtualization
 
             enable_nested_virtualization()
+        if config.DEPLOYMENT.get("enable_data_separation_replication"):
```

Comment:
Please document the `enable_data_separation_replication` parameter in conf/README.md file.